x-pack/plugins/endpoint/public/applications/endpoint/store/hosts/middleware.test.ts

```diff
@@ -56,7 +56,7 @@ describe('host list middleware', () => {
     await sleep();
     expect(fakeHttpServices.post).toHaveBeenCalledWith('/api/endpoint/metadata', {
       body: JSON.stringify({
-        paging_properties: [{ page_index: 0 }, { page_size: 10 }],
+        paging_properties: [{ page_index: '0' }, { page_size: '10' }],
```

Comment:
the page_index and page_size are strings because they are coming from the url now

Reply:
This does not seem valid. The Server Schema for the metadata POST request indicates these should be numbers. 
See https://github.com/elastic/kibana/blob/4ca86103b34c53e8da1e61a9aba3a77986070e64/x-pack/plugins/endpoint/server/routes/metadata/index.ts#L44

if you are sending strings to the server now (in the browser) and that is succeeding, then perhaps the platform translates it back to a number 🤷‍♂️